Admin/AdminBuilder.php

```diff
@@ -22,7 +22,7 @@
 /**
  * @author Christian Gripp <mail@core23.de>
  */
-final class AdminBuilder implements FormBuilderInterface, DatagridBuilderInterface
+final class AdminBuilder implements AdminBuilderInterface
```

Comment:
> What if I need to mock AdminBuilder, is there a way to generate a mock extending two interfaces?

Yes there is. phpunit standard mocking supports it, and maybe prophecy does too…